apis/network/v1alpha3/register.go

```diff
@@ -53,7 +53,17 @@ var (
 	SubnetGroupVersionKind = SchemeGroupVersion.WithKind(SubnetKind)
 )
 
+// SecurityGroup type Metadata.
+var (
+	SecurityGroupKind             = reflect.TypeOf(SecurityGroup{}).Name()
+	SecurityGroupGroupKind        = schema.GroupKind{Group: Group, Kind: SecurityGroupKind}.String()
+	SecurityGroupKindAPIVersion   = SecurityGroupKind + "." + SchemeGroupVersion.String()
+	SecurityGroupGroupVersionKind = SchemeGroupVersion.WithKind(SecurityGroupKind)
+)
+
 func init() {
 	SchemeBuilder.Register(&VirtualNetwork{}, &VirtualNetworkList{})
 	SchemeBuilder.Register(&Subnet{}, &SubnetList{})
+	SchemeBuilder.Register(&SecurityGroup{}, &SecurityGroupList{})
+	//SchemeBuilder.Register(&SecurityRule{})
```

Comment:
Should this be removed? Is it possible to create a `SecurityRule` independent of a `SecurityGroup`? If so, it should be a separate resource type.